db/migrate/20251204102032_remove_default_immunisation_import_type.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class RemoveDefaultImmunisationImportType < ActiveRecord::Migration[8.1]
+  def up
+    change_column_default :immunisation_imports, :type, nil
+  end
+
+  def down
+    change_column_default :immunisation_imports, :type, 0
+  end
```

Comment:
This has the same result, but reads nicer I think:

```suggestion
  def change
    change_column_default :immunisation_imports, :type, from: 0, to: nil
  end
```